service_config.go

```diff
@@ -136,7 +136,7 @@ type retryPolicy struct {
 	maxAttempts int
 
 	// Exponential backoff parameters. The initial retry attempt will occur at
-	// random(0, initialBackoffMS). In general, the nth attempt will occur at
+	// random(0, initialBackoff). In general, the nth attempt will occur at
 	// random(0,
 	//   min(initialBackoffMS*backoffMultiplier**(n-1), maxBackoffMS)).
```

Comment:
Can you fix `ms` in this line, too? Thanks!

Reply:
@menghanl Done. PTAL.

Reply:
@menghanl @easwars PTAL.